### PR TITLE
docs: add some badges in README

### PR DESCRIPTION
**Pull request by Dunqing**
### Description 

image

### Test Plan

---

### CODE REVIEW

**Comment by Dunqing** [comment]
* **#580** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/580?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Dunqing and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: README.md [hunk 1]
```diff
@@ -43,3 +55,19 @@ This project also partially contains code derived or copied from the following p
...
+[discord-badge]: https://img.shields.io/discord/1079625926024900739?logo=discord&label=Discord
+[discord-url]: https://chat.rolldown.rs
+[license-badge]: https://img.shields.io/badge/license-MIT-blue.svg
+[license-url]: https://github.com/oxc-project/oxc/blob/main/LICENSE
+[ci-badge]: https://github.com/rolldown/rolldown/actions/workflows/ci.yml/badge.svg?event=push&branch=main
+[ci-url]: https://github.com/rolldown/rolldown/actions/workflows/ci.yml?query=event%3Apush+branch%3Amain
+[npm-badge]: https://img.shields.io/npm/v/rolldown/latest?color=brightgreen
+[npm-url]: https://www.npmjs.com/package/rolldown/v/latest
+[code-size-badge]: https://img.shields.io/github/languages/code-size/rolldown/rolldown
+[code-size-url]: https://github.com/oxc-project/oxc
```
Comment (hyf0) [inline README.md]: I'm not sure if this is expected.

File: README.md [hunk 2]
```diff
@@ -43,3 +55,19 @@ This project also partially contains code derived or copied from the following p
...
+[code-coverage-badge]: https://codecov.io/github/rolldown/rolldown/branch/main/graph/badge.svg
+[code-coverage-url]: https://codecov.io/gh/rolldown/rolldown
+[playground-badge]: https://img.shields.io/badge/Playground-blue?color=9BE4E0
```
Comment (hyf0) [inline README.md]: I'm not sure if this is expected.

**Comment by Dunqing** [comment]
@Boshen cc, copy from oxc 😂

**Review by hyf0 — APPROVED** [review]

**Pull request merged**